backend/config/urls.py

```diff
@@ -18,19 +15,7 @@
     url(r"^admin/", admin.site.urls),
     url(r"^api/auth/", include("djoser.urls")),
     url(r"^api/auth/", include("djoser.urls.authtoken")),
+    url(r"^api/auth/", include("rest_framework.urls", namespace="rest_framework")),
     path(r"api/backend-status", views.index, name="index"),
     path("", include(router.urls)),
 ]
-
-urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
-urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
-
-# Browsable API login
-urlpatterns += [
-    url(r"^api/auth/", include("rest_framework.urls", namespace="rest_framework"))
-]
-
-if settings.DEBUG:

```

Comment:
Do we not want the debug tools anymore?

Reply:
I thought we weren't using it, but I tested it and it's very useful. Good catch again!
See commit #https://github.com/code-for-canada/project-thundercat/pull/308/commits/c1ef6926cea5e84ffe168323367ebb816dcd0ac4